authors/apps/articles/views.py

```diff
@@ -0,0 +1,69 @@
+from rest_framework.views import APIView
+from django.http.response import Http404
+from django.utils.text import slugify
+from rest_framework.response import Response
+from rest_framework import status
+from .serializers import ArticleSerializer
+from .models import Article
+
+
+class ArticleList(APIView):
+    """
```

Comment:
If you used generic views on this page you could have only a maximum of ten lines of code excluding the imports. You need to implement something like this here http://www.django-rest-framework.org/tutorial/3-class-based-views/#using-generic-class-based-views